source/slang/slang-ir-glsl-legalize.cpp

```diff
@@ -1285,6 +1285,33 @@ ScalarizedVal createSimpleGLSLGlobalVarying(
         declarator,
         &systemValueInfoStorage);
 
+    {
+        uint32_t index = 0;
+        IRSystemValueSemanticAttr* systemSemantic = nullptr;
+        for (; index < inVarLayout->operandCount; index++)
+        {
+            if (auto systemValueSemanticAttr = as<IRSystemValueSemanticAttr>(inVarLayout->getOperand(index)))
+            {    
+                systemSemantic = systemValueSemanticAttr;
+                break;
+            }
+        }
+        // Validate the system value, convert to a regular parameter if this is not a valid system value for a given target.
+        if (systemSemantic && isSPIRV(codeGenContext->getTargetFormat()) && systemSemantic->getName().caseInsensitiveEquals(UnownedStringSlice("sv_instanceid"))
+            && ((stage == Stage::Fragment) || (stage == Stage::Vertex && inVarLayout->usesResourceKind(LayoutResourceKind::VaryingOutput))))
+        {
+            IRCloneEnv cloneEnv;
```

Comment:
Instead of using `IRClone`, just use a `List` to store all the operands you want to retain, and create a new inst with that list of operands. Simply call `emitIntrinsicInst` to do that. There is no need to use the clone mechanim and the `cloneInstExcludingSomeOperands` function, since the layout inst will not have any children.